### DISCUSSION TITLE
Copilot4Eclipse 1.1  in MyEclipse 2023

### DISCUSSION THREAD

**Discussion by MMirabito** [Genuitec/Copilot4Eclipse-community#36] [Help] [answered]
Using **MyEclipse 2023** how can I use copilot **V1.1.0** when I go to the marketplace  I can only  **V1.2.0** which is compatible only with **MyEclipse 2024**

What can I do to install?

Thanks
max

**Comment by wayneparrott** [comment, marked as answer]
@MMirabito please send a note to support@genuitec.com. Include your MyEclipse subscriber ID and reference this thread. We will get you fixed up with Copilot4Eclipse 1.1 installer.

Wayne
Copilot4Eclipse Team

**Reply by MMirabito** [reply]
Thank you @wayneparrott  

max

**Comment by MMirabito** [comment]
All done working with [support@genuitec.com](mailto:support@genuitec.com) for further instructions

max
